src/js/components/Grommet/doc.js

```diff
@@ -37,9 +37,14 @@ export const doc = Grommet => {
       theme.`,
     ),
     userAgent: PropTypes.string.description(
-      `User agent used to detect the device width for setting the initial 
+      `User agent used to detect the device width for setting the initial
       breakpoint.`,
     ),
+    containerRoot: PropTypes.object.description(
+      `The node where Drop and Layer containers are inserted. Defaults to
+      document.body which is almost always the right choice. This is used
+      for less common cases like rendering within a shadow root.`,
```

Comment:
```suggestion
      for less common cases like rendering within an internal node (e.g shadow root).`,
```
Would this description make more sense? I'm trying to keep it as generic as possible.

Reply:
Looks good to me.

Reply:
Great, approve this fix and update the snapshots.